recipes/ua-nodeset/all/conandata.yml

```diff
@@ -0,0 +1,5 @@
+sources:
+  "1.04-2021-09-15":
```

Comment:
You have to use an official release. We only allow different names when there is no official release: https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#what-version-should-packages-use-for-libraries-without-official-releases

I understand this project is exceptional, but still we don't track branches, but releases. Please, revert to 1.02-

Reply:
To clarify:
1.04 is the current official release of the whole OPC UA NodeSets and Specification.

See for example here:
https://opcfoundation.org/developer-tools/specifications-unified-architecture/part-4-services/

You can see that the version of the specification is 1.04, i.e. the latest release.
This also matches with the Readme in UA-Nodeset repo:

> The files in the 1.04 branch are also public to the OPC Foundation website. If someone is looking for the officially released version of the UANodeSets they must follow the links in the specification.

The tags on the UA-Nodeset repo indicate specific versions of Sub Specifications.

E.g.,
`PADIM-1.02-2021-07-21` means that this tag includes all previously released Companion Specifications, but Updates the PADIM Specification to version 1.02.
Note that the PADIM Specification 1.02 still builds on top of the 1.04 Base Sepcification.

Yes I know that this is not straight forward, but unfortunately that's how the OPC Foundation decided to release their Companion Specs.

Reply:
yes, we will follow any release scheme upstream uses, doesn't matter how weird and complex is it

Reply:
Ok, all clear :)

So should we then use as the version `PADIM-1.02-2021-07-21` (i.e., the latest tag of the 1.04 branch)?

So the conan package will be `ua-nodeset/PADIM-1.02-2021-07-21`?

Just using `ua-nodeset/1.02-2021-07-21` would not be correct, as one would assume that it is the 1.02 companion specification, and not 1.04 (which is included in the PADIM-1.02-2021-07-21).

I just changed the recipe to that tag.

Reply:
I am not the one who judge - you probably know better which version(s) you need.
we usually prefer latest official releases, unless there are strong reasons not to use them (e.g. breaking API changes). 
if latest tag is latest release in that repository, than okay.
their versioning scheme is super confusing for the outsiders, so I would use tag name as is.
at least to distinguish base versions from PADIM versions (I have no idea what PADIM is and what's the difference from base, especially from the consumer point of view).

Reply:
Yep, I completely agree that their tagging is very confusing.

IMHO using the Tag Name as version is less confusing from our side, than inventing a custom version string for conan.

SoIwpuld recommend to just use the complete tag, as it currently is 👍🏼